src/app/routes/routes.js

```diff
@@ -217,13 +219,6 @@ module.exports = (Router, Service, App) => {
     }
 
     return Service.User.RegisterUser(req.body)
-      .then((result) => {
-        if (req.body.referrer) {
-          Logger.warn('Register for %s by referrer %s', result.user.email, req.body.referrer);
-          return Service.AppSumo.ApplyLicense(result.user, req.body.referrer).then(() => result);
```

Comment:
Check this

Reply:
Is this param sent from Drive-Web @carlossalasamper? This could break the space applied to AppSumo users (and sadly, non-AppSumo users too)